### PR TITLE
Main Dashboard

### PR DESCRIPTION
**Pull request by greyluo**
- For now, the create button will create a new note and store it into local storage that we use for testing. Its function should be changed later. 
- We also use localStorage for storing and retrieving notes from the database for testing the front-end. It should also be updated.

### CODE REVIEW

File: source/scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ * append the list items to the list in the document
+ * @param {Object} notes containing all the notes in the local storage
+ */
+function addNotesToDocument(notes) {
+    let list = document.querySelector('.list');
+
+    notes.forEach(note => {
+        let listItem = document.createElement('list-item');
```
Comment (alexzhang1618) [inline source/scripts/index.js:42]: Can we use a more descriptive name than 'list-item'? Maybe something like 'dashboard-row' might be a little more clear that this element is meant for the dashboard

File: source/scripts/listItem.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+class listItem extends HTMLElement{
```
Comment (alexzhang1618) [inline source/scripts/listItem.js:1]: Here too

**Review by alexzhang1618 — COMMENTED** [review]
Great work! Thanks for moving everything to the new structure. One small comment/suggestion:

File: source/scripts/index.js (hunk 1 above)
Comment (greyluo) [reply source/scripts/index.js]: Should be good now

File: source/scripts/index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ * append the new row to the dashboard in the document
+ * @param {Object} notes containing all the notes in the local storage
+ */
+function addNotesToDocument(notes) {
+    let dashboard = document.querySelector('.dashboard');
+
+    notes.forEach(note => {
+        let row = document.createElement('dashboard-row');
```
Comment (priyakumar0817) [inline source/scripts/index.js:42]: Good, changed name of dashboard to a more descriptive variable name.

**Review by priyakumar0817 — APPROVED** [review]
Good, script files (dashboardRow.js and index.js) have descriptive comments to let the easily follow code. Will approve merging these changes now, one suggestion is to add any comments in your index.css file that may help future readers or anyone who may work on it. Otherwise good for now!

**Pull request merged**